platforms/quorum/configuration/roles/create/crypto/constellation/tasks/nested_main.yaml

```
@@ -30,3 +36,4 @@
     VAULT_ADDR: "{{ vault.url }}"
     VAULT_TOKEN: "{{ vault.root_token }}"
   when: vault_tm_result.failed == True
+  ignore_errors: yes
```

Why is ignore error added here? This should fail if the vault cannot be updated